packages/scripts/src/admin-proposals/registerContract.js

```diff
@@ -424,10 +425,10 @@ async function run() {
               fromBlock: 0,
             }
           );
-          const relayedRegisterContractEvent = relayedTransactions.find(
-            (e) =>
-              e.returnValues.calls ===
-              [{ to: contractsByNetId[network.chainId].registry.options.address, data: registerContractData }]
+          const relayedRegisterContractEvent = relayedTransactions.find((e) =>
```

Comment:
The newer typescript version errored to build this as it does not allow comparison of object references